traffic_router/core/src/main/conf/log4j2.xml

```diff
@@ -18,14 +18,18 @@
         <Console name="stdout" target="SYSTEM_OUT">
             <PatternLayout pattern="%m%n" />
         </Console>
-        <RollingFile name="traffic_router_access" fileName="/opt/traffic_router/var/log/access.log" >
+        <RollingFile name="traffic_router_access"
+                     fileName="/opt/traffic_router/var/log/access.log"
+                     filePattern="/opt/traffic_router/var/log/access-%d{yyyyMMdd}.log" >
```

Comment:
`${deploy.dir}` is used in `core/src/test/conf/log4j2.xml` instead of `/opt/traffic_router`:
https://github.com/apache/trafficcontrol/blob/79ef73dc75f27a0a39af5ce6382760a1143b65cb/traffic_router/core/src/test/conf/log4j2.xml#L32-L33

Can it be used here, too?